core/controllers/reader.py

```diff
@@ -44,43 +44,15 @@
 
 MAX_SYSTEM_RECOMMENDATIONS = 4
 
-SHARING_OPTIONS = config_domain.ConfigProperty(
-    'sharing_options', {
-        'type': 'dict',
-        'properties': [{
-            'name': 'gplus',
-            'schema': {
-                'type': 'bool',
-            }
-        }, {
-            'name': 'facebook',
-            'schema': {
-                'type': 'bool',
-            }
-        }, {
-            'name': 'twitter',
-            'schema': {
-                'type': 'bool',
-            }
-        }]
-    },
-    'Sharing options to display in the learner view',
-    default_value={
-        'gplus': False,
-        'facebook': False,
-        'twitter': False,
-    })
-
 SHARING_OPTIONS_TWITTER_TEXT = config_domain.ConfigProperty(
-    'sharing_options_twitter_text', {
+    'sharing_options_twitter_text_reader', {
```

Comment:
I decided to rename this to DEFAULT_TWITTER_SHARE_MESSAGE_PLAYER -- it seems like a clearer name.